packages/mdc-ripple/_mixins.scss

```diff
@@ -179,8 +192,12 @@
     &#{$pseudo} {
       top: 0;
       left: 0;
+
+      /* @alternate */
       width: $radius;
       width: var(--mdc-ripple-fg-size, $radius);
+
+      /* @alternate */
       height: $radius;
       height: var(--mdc-ripple-fg-size, $radius);
       transform: scale(0);
```

Comment:
Interesting. I guess it just feels weird to have a blank line before `/* alternate */` (which has the effect of grouping duplicate properties together) but _not_ a blank line before `transform`.